modules/ListView.js

```diff
@@ -295,7 +300,11 @@
           // Shouldn't get here. Item view is missing...
         }
       }, this);
+      renderPromises.push(this.attachTrackedViews() || $.Deferred().resolve().promise());
+      this.__injectionSiteMap = {};
+      renderPromises.push(this.postrender() || $.Deferred().resolve().promise());
       this.trigger('render-complete');
+      return $.when.apply($, _.flatten(renderPromises));
```

Comment:
$.when takes a comma separately set of arguments. I have an array. I use apply to convert an array into separate arguments